### DISCUSSION TITLE
Openclash

### DISCUSSION THREAD

**Discussion by ivanors** [helmiau/HelmiWrt-OS#38] [Q&A]
Bang Helmi, Openclash mode rule kok bengong ya? kalau mode global juga kurang responsif

**Comment by helmiau** [comment]
> Bang Helmi, Openclash mode rule kok bengong ya? kalau mode global juga kurang responsif

aman bang, udah pake clash core yg premium kok kayak yg punya om reyre, bahkan versi terbarunya
